1. Basic Web Development/server/package.json

```diff
@@ -22,7 +22,6 @@
     "fs-extra": "^10.0.0",
     "kill-port": "^1.6.1",
     "nodemon": "^2.0.12",
-    "npm-run-all": "^4.1.5",
```

Comment:
You removed this dependency but the scripts still seem to be using it. 
Alternatively you could do sth like this and drop it altogether. 
`"build": "npm run clean && npm run lint && np run tsc",`